calendar_event_multi_company/security/calendar_event.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
```

Comment:
```suggestion
<odoo noupdate="1">
```

In Odoo, Ir.rules are set as no updatable, so it can be changed by UI.

Reply:
Not always. And I prefer to let them as noupdate=0, for not having to deal with migration scripts. At the end, if you are installing this module, it's because you want this behavior set by code. If not, create your own record rule.

Reply:
Make sense ! I thought it was an OCA convention.